modules/k8s-nmstate-troubleshooting-dns-disconnected-env.adoc

```diff
@@ -6,7 +6,7 @@
 [id="troubleshooting-dns-disconnected-env_{context}"]
 = Troubleshooting DNS connectivity issues in a disconnected environment
 
-If you experience DNS connectivity issues when configuring `nmstate` in a disconnected environment, you can configure the DNS server to resolve the list of name servers for the domain `root-servers.net`.
+If you experience DNS connectivity issues when configuring `nmstate` in a disconnected environment, you can configure the DNS server to resolve the list of name servers for the domain `root-servers.net`. Alternatively, you can resolve DNS connectivity issues by configuring a custom host name and then using the hostname to perform internal networking health checks on your cluster.
```

Comment:
Rather

>If you experience **health check probe issues** when configuring `nmstate` in a disconnected environment, you can configure the DNS server to resolve **the custom domain name instead of the default** domain `root-servers.net`.

Reply:
:heavy_check_mark: